lib/customerio.ex

```diff
@@ -801,4 +801,24 @@ defmodule Customerio do
     {:ok, result} -> result
     {:error, e} -> raise e
   end
+
+  @doc """
+  Trigger a campaign execution.
+
+  ## Params
+
+    * `id` - the unique identifier of the campaign.
+
+    * `data` - data to be included in the campaign.
+
+  ## Example
+
+  ```elixir
+  iex> Customerio.trigger_campaign(1, %{data: %{title: "hello"}})
+  ```
+  """
+
+  def trigger_campaign(id, data \\ []) do
```

Comment:
API has additional data, but data seems to be a map, not a list(Keyword).
May be it'll be great idea to add a test with additional params, to ensure that they are passed in a right way.

Also, each function in library has opportunity to controll hackney with additional params.
Let's keep this opportunity.

```suggestion
  def trigger_campaign(id, data_map, opts \\ []) do
```